cmd/setup-node/commands/root.go

```diff
@@ -64,7 +64,7 @@ var rootCmd = &cobra.Command{
 
 		sn, err := setup.NewNode(conf, metrics.NewPrometheus("setupnode"))
 		if err != nil {
-			logger.Fatal("Failed to setup Node: ", err)
+			logger.Fatal("Failed to setup Visor: ", err)
```

Comment:
It should be `"Failed to setup a setup node: "`, but I'd go with `"Failed to create a setup node: "`